src/QsCompiler/Tests.Compiler/ExecutionTests.fs

```diff
@@ -143,7 +143,7 @@ type ExecutionTests(output: ITestOutputHelper) =
     member this.``Referencing Projects and Packages``() =
         ExecuteAndCompareOutput "PackageAndProjectReference" "
                 Welcome to Q#!
-                Info: Go check out https://docs.microsoft.com/en-us/quantum/?view=qsharp-preview.
+                Info: Go check out https://docs.microsoft.com/azure/quantum/?view=qsharp-preview.
```

Comment:
Might drop `?view=qsharp-preview` from the URL, since it's not used any longer. But I'm not sure it's worth another round of edits

Reply:
Good point, removed that part.